server/tournaments/index.ts

```diff
@@ -1,5 +1,6 @@
 import {Elimination} from './generator-elimination';
 import {RoundRobin} from './generator-round-robin';
+import {LastTournamentDataNode} from '../rooms';
```

Comment:
This should be `type LastTournamentDataNode = import(...`

Reply:
```suggestion
type LastTournamentDataNode = import('../rooms').LastTournamentDataNode;
```

Reply:
Got it, thanks!